packages/fxa-settings/src/components/Settings/PageSecurityEvents/index.test.tsx

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import 'mutationobserver-shim';
+import '@testing-library/jest-dom/extend-expect';
+import { act, screen } from '@testing-library/react';
+import { renderWithRouter, mockAppContext } from '../../../models/mocks';
+import React from 'react';
+import PageSecurityEvents from '.';
+import { Account, AppContext } from '../../../models';
+import { MOCK_SECURITY_EVENTS } from './mocks';
+import { logViewEvent, settingsViewName } from '../../../lib/metrics';
+
+const account = {
+  primaryEmail: {
+    email: 'foxy@mozilla.com',
+  },
+  getSecurityEvents: jest.fn().mockResolvedValue(MOCK_SECURITY_EVENTS),
+} as unknown as Account;
+
+const render = (acct: Account = account) =>
+  renderWithRouter(
+    <AppContext.Provider value={mockAppContext({ account: acct })}>
+      <PageSecurityEvents />
+    </AppContext.Provider>
+  );
+
+describe.only('Security Events', () => {
+  it('renders', async () => {
+    await act(async () => {
+      await render();
+    });
+    expect(screen.getByTestId('flow-container')).toBeInTheDocument();
+    expect(screen.getByTestId('flow-container-back-btn')).toBeInTheDocument();
+
+    expect(account.getSecurityEvents).toBeCalled();
+
+    expect(
+      screen.getByTestId('security-events-account-create')
+    ).toBeInTheDocument();
+    expect(
+      screen.getByTestId('security-events-account-disable')
+    ).toBeInTheDocument();
+    expect(
+      screen.getByTestId('security-events-account-enable')
+    ).toBeInTheDocument();
+    expect(
+      screen.getByTestId('security-events-account-login')
+    ).toBeInTheDocument();
+    expect(
+      screen.getByTestId('security-events-account-reset')
+    ).toBeInTheDocument();
+    expect(
+      screen.getByTestId('security-events-emails-clearBounces')
+    ).toBeInTheDocument();
```

Comment:
With the new React refactor stuff we've been pushing to use some better selectors in our tests, you could use these instead and potentially even remove the data-test ids:

```
screen.getByRole('heading', {
  name: 'Recent Account Activity',
});
screen.getByText('Account was created');
screen.getByText('Account was disabled');
// etc.
```